### PR TITLE
Beta Project Viewer(Part 1)

### PR DESCRIPTION
**Pull request by noahmalmed**
# Description 
Completing the first part of #161

This PR includes every part of the workflow up to the actual swipe classifier, which will come in a follow up PR. 

# Details
- Add code to change nav bar color
- Add code to load beta projects
- Add code to display beta projects on discipline page and project page
- Update some design
- Update unit tests

# Review Checklist

- [x] Does it work in Android and iOS?
- [x] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [x] Are tests passing?

### CODE REVIEW

**Comment by wgranger** [comment]
I'm getting some failing tests. Make sure your client is set for the correct environment (instead of switching to 'production' on the client module). There are 11 tests that fail.

**Comment by noahmalmed** [comment]
Hmm I'm not seeing anything after doing a delete node_modules and a reinstall.

I'm guessing something is wrong with your setup because CI tests are also passing.

**Comment by noahmalmed** [comment]
@wgranger this should do the trick

File: src/components/NavBar.js [hunk 1]
```diff
@@ -164,7 +165,14 @@ const styles = EStyleSheet.create({
   },
   disabledIcon: {
     color: '$transparent'
-  }
+  },
+  totalClassifications: {
+    color: '$darkTextColor',
+  },
+  userName: {
```
Comment (wgranger) [inline src/components/NavBar.js]: Is this, and the style above, being used in this component?

File: src/components/ProjectDisciplines.js [hunk 2]
```diff
@@ -28,13 +28,15 @@ const mapStateToProps = (state) => ({
   isGuestUser: state.user.isGuestUser,
   isConnected: state.main.isConnected,
   isFetching: state.main.isFetching,
-  pushPrompted: state.user.pushPrompted
+  pushPrompted: state.user.pushPrompted,
```
Comment (wgranger) [inline src/components/ProjectDisciplines.js]: Not sure this is being used. Instead, line 64 uses `this.props.user.pushPrompted`, which seems redundant.

File: src/components/projects/ProjectList.js [hunk 3]
```diff
@@ -73,22 +85,36 @@ class ProjectList extends Component {
...
-        if (this.props.swipeEnabledProjects.length > 0) {
-            sections.push({data: this.props.swipeEnabledProjects, title: 'Made For Mobile'});
-        }
-
-        if (this.props.nonSwipeEnabledProjects.length > 0) {
-            sections.push({data: this.props.nonSwipeEnabledProjects, title: 'In-Browser Experience'})
+        if (inTestMode) {
+            if (!R.isEmpty(ownerIds)) {
+                const ownerProjects = swipeEnabledProjects.filter((project) => ownerIds.includes(project.id))
+                sections.push({data: ownerProjects, title: 'You Projects'})
```
Comment (wgranger) [inline src/components/projects/ProjectList.js]: Your projects?

**Comment by wgranger** [comment]
screen shot 2018-03-29 at 12 07 19 pm

This is the view on an iPhone SE. Notice how the text wraps onto the second line and looks a bit odd. I can't find exactly in this PR where this difference would've been made, but it isn't happening on an SE with the current version of the app.

**Comment by wgranger** [comment]
Nice. Looks good.

**Pull request merged**